### ISSUE TITLE
MBP: My Trip Report display gives error only for Loren

### ISSUE THREAD

**Issue by WaBirder57** [washingtontrails/vms#465]
Loren reports (http://staging.wta.org/@@backpacks/scrnm-Loren%20Drummond): Running across a strange error message. 

Logged in, from my dashboard, clicking My Trip Reports tile, I get an error message. (Screenshot attached). [Note this is low priority - don't hold release on this  as it seems to ONLY apply to Loren (see below on thoughts why]

Have cleared my cache, and get the error across FF and Chrome. Kindra is not getting the error message. Another visitor to my account can also see my trip report display fine, so it appears to only be my view of it, logged in. 

Error on this page: 
http://staging.wta.org/@@backpacks/scrnm-Loren%20Drummond/trip-reports

This view (logged out, or as another person viewing my reports) works:
http://staging.wta.org/go-hiking/trip-reports/tripreport_search?author=loren

[Note that at some point this year we were cleaning up the plone account database and Loren had an error where she mapped many other users Trip Reports to her Plone account.  She thinks this may be related to that situation and that in her Plone information for those specific trip reports have an error around her screen name which has a space in it.]

![mbplisttripreporterror](https://cloud.githubusercontent.com/assets/8312206/11578199/09efee46-99d8-11e5-8a4d-88acf0dbf6c4.JPG)

**Comment by davisagli** [comment]
I agree this looks like something wrong with a particular trip report. We'll take a look sometime after launch.
